packages/backend/src/tokens/request.ts

```diff
@@ -188,6 +203,53 @@ ${error.getFullMessage()}`,
     return signedOut(authenticateContext, reason, message);
   }
 
+  // NOTE(izaak): I don't love that null return implies signed-in...
```

Comment:
Another  option is I could take _and_ sometimes return a `SignedInState`, which also feels bad. I'll think about how to improve the structure, but ideas welcome.

Reply:
Let's leave as `null` but clarify on some JSDocs!